acceptance_tests/features/steps/view_collection_exercise_live_state.py

```diff
@@ -39,14 +38,7 @@ def go_live_date_hits(_, survey, period):
 @then('the state of a collection exercise is to be changed to Live')
 @then('they are able to see the Live Status for that Collection Exercise')
 def ce_details_state_is_live(_):
-    ce_state = collection_exercise_details.get_status()
-    for i in range(20):
-        if collection_exercise.is_live(ce_state):
-            break
-        time.sleep(1)
-        browser.reload()
-        ce_state = collection_exercise_details.get_status()
-    assert collection_exercise.is_live(ce_state), ce_state
+    assert is_text_present_with_retry('Live', 2)
```

Comment:
Do we need more retries here as it was up to 20 before suggesting it might be something that's slow to process it

Reply:
20?  I just used the numbers that were in the initial PR.  No problems in increasing that though